chart/templates/mayastor/obs/stats-service.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.eventing.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}-stats-aggregator
+  labels:
+    app: stats-aggreagtor
+    {{ include "label_prefix" . }}/release: {{ .Release.Name }}
+    {{ include "label_prefix" . }}/version: {{ .Chart.Version }}
+spec:
+  ports:
+    - name: stats
+      port: 9505
+      targetPort: 9505
+      protocol: TCP
+  selector:
+    app: stats-aggregator
+    {{ include "label_prefix" . }}/release: {{ .Release.Name }}
+{{- end }}
```

Comment:
Don't we need a customisable type? Is it always going to be ClusterIP?

ref:- https://github.com/openebs/mayastor-extensions/blob/develop/chart/templates/mayastor/apis/api-rest-service.yaml

Reply:
As far as i can recall , it was stated that we use cluster ip only.

Reply:
It should be customizable IMO.

Reply:
I think should be customisable with cluster ip as default. But I will leave upto you.

Reply:
Addressed your comment.